ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -22,6 +23,14 @@ import "./big-buttons";
 import "./track-activity";
 import "./vrmode";
 import cx from "classnames";
+// @ts-ignore
+import airplay from "@silvermine/videojs-airplay";
+// @ts-ignore
+import chromecast from "@silvermine/videojs-chromecast";
+import "@silvermine/videojs-chromecast/dist/silvermine-videojs-chromecast.css";
```

Comment:
I believe these two CSS imports should be able to go in `styles.scss`: both plugins have a bit in their readmes about SCSS ([here](https://github.com/silvermine/videojs-airplay#using-the-css-and-images) and [here](https://github.com/silvermine/videojs-chromecast#using-the-css-and-images)).

Reply:
I went ahead and moved the imports to the SCSS file.